lib/model/QubitTerm.php

```diff
@@ -928,4 +928,21 @@ public function getTreeViewSiblings(array $options = array())
 
     return $results;
   }
+
+  /**
+   * Get terms by taxonomy id. This function gets its results from ElasticSearch.
+   */
+  public static function getTermsByTaxonomyId($taxonomyId, $limit = 10)
```

Comment:
I think it's better to use a more specific name for this function, like 'getEsTermsByTaxonomyId'